crawler/collector.go

```diff
@@ -24,26 +25,21 @@ func (collector *URLCollector) sync(f func()) {
 	collector.Unlock()
 }
 
-func (collector *URLCollector) Visited(m map[uint64]bool, s string) (visited bool) {
+func (collector *URLCollector) visited(m map[uint64]bool, s string) (visited bool) {
```

Comment:
Does method starting with small caps char private?

Reply:
Yeah.

Reply:
Was it previously being used as publically and now not being used as publically?

Reply:
There is no public usage. That's why I lowercased it.